### DISCUSSION TITLE
/service_relation_server_resp_time/values[0]/id

### DISCUSSION THREAD

**Discussion by dengyichao** [apache/skywalking#9382] [Q&A]
9.0.0 version 
click General Service -> Topology errors

The field at path '/service_relation_server_resp_time/values[0]/id' was declared as a non null type, but the code involved in retrieving data has wrongly returned a null value. The graphql specification requires that the parent field be set to null, or if that is non nullable that it bubble up null to its parent and so on. The non-nullable type is 'ID' within parent type 'KVInt' The field at path '/service_relation_server_cpm/values[0]/id' was declared as a non null type, but the code involved in retrieving data has wrongly returned a null value. The graphql specification requires that the parent field be set to null, or if that is non nullable that it bubble up null to its parent and so on. The non-nullable type is 'ID' within parent type 'KVInt'

**Comment by Superskyyy** [comment]
This is fixed in V9.1 I believe, along with many more fixes. I suggest you to update to v9.1.
